package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@mongodb-js/saslprep": "^1.1.5",
-    "bson": "^6.5.0",
+    "bson": "github:mongodb/js-bson#NODE-6059-cleanup",
```

Comment:
TBD, right?

Reply:
Yes, we're working on finalizing a bson release, and we'll put that here before merging.

Reply:
Updated BSON version